README.md

```diff
@@ -149,28 +145,68 @@ $ bundle exec rake alchemy:install
 
 and follow the on screen instructions.
 
+### Render Alchemy Content in Solidus Layout
+
+If you plan to render the Alchemy site in the Solidus layout add the following
+to your initializer:
+
+```ruby
+# config/initializers/alchemy.rb
+require 'alchemy/solidus/alchemy_in_solidus'
+```
+
 ### Render Alchemy Content in Solidus views
 
-If you plan to render Alchemy content in your Solidus views (ie. a global header or footer section), you need to include the Alchemy view helpers and language store in your Solidus controllers.
+If you plan to render Alchemy content in your Solidus views (ie. a global header
+or footer section), you need to include the Alchemy view helpers and language
+store in your Solidus controllers with the following addition to your
+initializer:
 
 ```ruby
-# config/initializers/solidus.rb
-...
-Spree::BaseController.class_eval do
-  include Alchemy::ControllerActions
-end
+# config/initializers/alchemy.rb
+require 'alchemy/solidus/alchemy_in_solidus'
```

Comment:
Good catch. When splitting the commits I must have gotten things moved around. I have uploaded new commits that should be right and still is spit up some to allow better review.